### PR TITLE
fix: file location based pat for `bot_env_vars.env` and `../db/prod/*`

### PR DESCRIPTION
**Pull request by dataCobra**
Hey @bastienboutonnet,

Yesterday in the stream you encountered the problem that you had to rename the chatbot.py file and a lot of messy changes to imports were necessary to get the chatbot working again.

**Today I found what the problem was.**

--> Besides the PATH change for `../db/prod/`, you also had to adjust the path for `bot_env_vars.env`.

The change I present you here is live tested with my own bot. :+1:

I used `black` (of course) to refactor both files. :slightly_smiling_face:

Regards,
Ben

### CODE REVIEW

**Comment by dataCobra** [comment]
Well, it seems like your pre-commit doesn't like my refactoring with black... :disappointed:

File: chatbot/config.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from dotenv import load_dotenv
 
-load_dotenv("bot_env_vars.env")
+load_dotenv(os.path.join(os.path.dirname(__file__), "bot_env_vars.env"))
```
Comment (bastienboutonnet) [inline chatbot/config.py:5]: 👌

File: chatbot/db.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # TODO: we might want to have a list of available commands somewhere in the class so that we can
 # quickly check before updating so that we don't crash.
 class DbConnector:
-    def __init__(self, db_path: str = "../db/prod/"):
+    def __init__(self, db_path: str = os.path.join(os.path.dirname(__file__), "../db/prod/")):
```
Comment (bastienboutonnet) [inline chatbot/db.py:17]: yeah we did that on strream already but since the PR for the badges is still unfinished I'm welcoming this change early.

**Review by bastienboutonnet — APPROVED** [review]
Hey @dataCobra thanks a lot for the changes. Some of these are already in #14 but I prefer your way of specifying the path for the env vars file so I'll definitely approve and merge this PR

**Pull request merged**